test/TestProjects/MgmtLRO/tests/MgmtLROManagementTestBase.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I feel like we don't change any of the existing tests. You shouldn't change any default behavior for generating tests/samples.

Only when the new project configuration is turned on, there will be new test base files and sample projects generated.

Reply:
That being said, you do a regen on azure-sdk-for-net. nothing should change.